pom.xml

```diff
@@ -12,6 +12,7 @@
 		<graphdb.version>9.9.0</graphdb.version>
         <kafka.version>2.4.0</kafka.version>
         <jackson.version>2.10.1</jackson.version>
+		<slf4j.version>1.7.30</slf4j.version>
```

Comment:
fix formatting + I am not sure you actually need to add slf4j as a separate dependency